runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1323,7 +1351,7 @@ sub startRun {
 								} else {
 									$metricsStr = ", $successStr, throughput:$tptStr, avgRT:$rtStr";									
 								}
-								$console_logger->info("   [workload $workloadNum, appInstance: $appInstanceNum] Ended: $nameStr${metricsStr}.");
+								$console_logger->info("   [$tempOutputWorkloadNum appInstance: $appInstanceNum] Ended: $nameStr${metricsStr}.");
```

Comment:
This results in "[ appInstance"  (with a space) in some cases.
Could probably move that space to line 1270 in the then part  following the comma where the $tempOutputWorkloadNum is set?